lambda/getBasicInformationLambda.ts

```diff
@@ -110,7 +115,11 @@ export const handler = async (
 			expires: 300,
 		})({ measurements })
 	}
+	const dataUsagePerTimespan = await getDataUsagePerTimespanFunc(iccid)
```

Comment:
**refactor**: Inline this function here. It then also becomes more clear what the data structure is that you are are returning.

Reply:
That means that we don't need a test for that function right?

Reply:
No, it's 4 calls to the the function that creates the query (which is tested) and then calls to functions which are not part of our source, so an extra test is not needed.

Reply:
No, as in, we do not need a test.